compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/window/dialog/DialogTest.kt

```diff
@@ -65,7 +65,8 @@ class DialogTest {
 
             fun createWindow() = ComposeDialog().apply {
                 size = Dimension(300, 200)
-
+                isResizable = true
+                isUndecorated = true
```

Comment:
Do we need this change in DialogTest?

Reply:
It covers this line of code [androidx/compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/UndecoratedWindowResizer.desktop.kt](https://github.com/JetBrains/androidx/blob/e529c7e79bee073a21bc789f2dc5b10b60e4dddd/compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/UndecoratedWindowResizer.desktop.kt#L69)

Reply:
So we can catch in unit test potential bugs

Reply:
Ack